src/pages/home/base/mobile-core/lifecycle/index.md

```diff
@@ -12,40 +12,203 @@ import Tabs from './tabs/index.md'
+
+You should pause Lifecycle collection when the user stops using your app. The best time to do this is usually when your app has entered the background.
+
+### Lifecycle on iOS
+
+#### Start Lifecycle data collection on launch
+
+Start Lifecycle data collection by calling `lifecycleStart(_:)` from within the callback of the `MobileCore.registerExtensions(_:)` method in your app's `application(_:didFinishLaunchingWithOptions:)` delegate method.
+
+If your iOS application supports background capabilities, your `application(_:didFinishLaunchingWithOptions:)` method might be called when iOS launches your app in the background. If you do not want background launches to count towards your lifecycle metrics, then `lifecycleStart(_:)` should only be called when the application state is not equal to `UIApplicationStateBackground`.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Swift
+
+<Tabs query="platform=ios-swift&task=start-lifecycle-didfinishlaunch"/>
+
+Objective-C
+
+<Tabs query="platform=ios-objc&task=start-lifecycle-didfinishlaunch"/>
+
+#### Start and Pause Lifecycle data collection from iOS lifecycle delegate
+
+When your app is resuming from the background state, call `lifecycleStart(_:)` from the appropriate delegate object's "will enter foreground" method. When your app enters the background state, call `lifecyclePause()` from the appropriate delegate object's "did enter background" method.
+
+* For a scene-based UI, call the Lifecycle APIs from the _UISceneDelegate_'s `sceneWillEnterForeground(_:)` and `sceneDidEnterBackground(_:)` methods.
+
+* For all other apps, call the Lifecycle APIs from the _UIApplicationDelegate_'s `applicationWillEnterForeground(_:)` and `applicationDidEnterBackground(_:)` methods.
+
+* If your application supports both a scene delegate and an app delegate, implement the Lifecycle APIs in both delegate objects.
+
 <TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
 
-Android
+Swift
+
+<Tabs query="platform=ios-swift&task=start-pause"/>
+
+Objective-C
 
-<Tabs query="platform=android&task=add"/>
+<Tabs query="platform=ios-objc&task=start-pause"/>
 
-iOS
+<InlineAlert variant="info" slots="text"/>
+
+For more information on handling foreground and background states in applications with Scenes, refer to Apple's documentation on preparing your UI to run in the [foreground](https://developer.apple.com/documentation/uikit/app_and_environment/scenes/preparing_your_ui_to_run_in_the_foreground) and [background](https://developer.apple.com/documentation/uikit/app_and_environment/scenes/preparing_your_ui_to_run_in_the_background)
+
+#### Start and Pause Lifecycle data collection in SwiftUI
+
+If your pure SwiftUI application does not use an app delegate or scene delegate, you may still use the Lifecycle extension by listening for _scenePhase_ changes.
```

Comment:
Is "SwiftUI" one word? Just want to double check on that.

Reply:
Yes. It's the name of a product. https://developer.apple.com/xcode/swiftui/